lib/kontena/machine/vagrant/common.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+require 'erb'
+require 'open3'
+require 'json'
+
+module Kontena
+  module Machine
+    module Vagrant
+      module Common
+
+        def erb(template, vars)
+          ERB.new(template).result(OpenStruct.new(vars).instance_eval { binding })
+        end
+
+        def run_command(cmd)
+          exit_code = 0
+          Open3.popen3(cmd) do |stdin, stdout, stderr, wait|
```

Comment:
If this just passes through stdout/stderr without caring about the output, why not just use `system`, and exec the process directly with the original stdout/stderr? Wouldn't need any thread for that.

Also, popen with a stdin pipe that just gets ignored doesn't sound sensible either... vagrant would just block if it tries reading from it?